### DISCUSSION TITLE
SEQ not running on http://localhost:5341/

### DISCUSSION THREAD

**Discussion by Verbient** [datalust/seq-tickets#1771] [Help]
Hi, I installed SEQ today and tried to setup through the wizard in my AWS windows machine. I tried configuring SSL through the thumbprint, but it did not work. So I uninstalled SEQ from Control Panel and reinstalled it. After reinstalling it would not allow me to use the wizard and I get the [error](https://prnt.sc/2nqBl-xmfnP0) 
I tried deleting the ProgramData/SEQ folder and reinstalled, still it did not work.

Further seq is not running on "http://localhost:5341/" Please advise.

PS: You should have created few youtube videos to help user with the installation.

**Comment by liammclennan** [comment]
Hi @Verbient ,

The dialog you shared is a message not an error. Try continuing the wizard. As you have uninstalled Seq it will not be running at `http://localhost:5341` until you complete the installation. 

Sorry about the lack of Youtube videos, but we do have [documentation of the installation process with screenshots](https://docs.datalust.co/docs/getting-started), and documentation for [setting up TLS/SSL](https://docs.datalust.co/docs/ssl). 

If none of that solves the problem for you please send any error messages you encounter.

**Reply by Verbient** [reply]
I reinstalled it successfully. But still localhost:5341 is not working in my windows AWS machine. What am I doing wrong? Please check the  [screenshot](https://prnt.sc/ntaT7kMWn3ba). I tried configuring my custom domain seq.mydomain.com and it was unsuccessful too. 
I don't need any help in installation. There is nothing challenging about installation. Secondly SSL comes into question only when I am able to configure it and get it running in the first place. I have tried running all commands to stop, uninstall, reinstall; still nothing works.

**Reply by liammclennan** [reply]
As you have deployed Seq in AWS you will need to ensure that network access is open between wherever you are requesting from and the server. For example, the AWS security group and possibly the local firewall on the server will need to allow inbound connections on port `5341`.

**Reply by Verbient** [reply]
Port is enabled, yet issue is unresolved

**Reply by nblumhardt** [reply]
At an administrative PowerShell console on the server machine, `seq service status` should tell you whether Seq is running.

If it is, running `seq config get -k api.listenUris` should show you what URL Seq is listening on. You can send these details through to us via `support@` if you need us to take a look.

Hope this helps!

**Reply by Verbient** [reply]
Thanks Nicholas, I tried re-installing SEQ with my custom domain, however I got the following [Error](https://prnt.sc/Z0ifhwQ3r93X)

Then I tried running the command 'seq config get -k api.listenUris' and it returned me 'http://seq.xxxx.com/' 

After restarting the server the following [error](https://prnt.sc/Z0ifhwQ3r93X) is gone. I was able to successfully install SEQ in 'http://seq.xxxx.com'. but unfortunately still the problem persist and I am unable browse to the url. 

I again uninstalled and tried using default options 'localhost...' and I got an error message "URL reservation delete failed, Error: 2 The system cannot find the file specified. Url reservation add failed, Error: 183
Cannot create a file when that file already exists. Could not add URL reservation for http://+:5341/: `netsh` returned 1; ignoring"

**Reply by Verbient** [reply]
I was able to figure the solution myself. I had to create a DNS A-record for the subdomain seq.xxxx.com. Can you please add this step in your documentation. If you remember Liam, an year back I faced similar issue and you amended your documentation and added the word **MACHINE** in the syntax: 
`seq service install --listen="https://seq.example.com" -u "MACHINE\user" -p "password"`

Now I am facing another challenge in enabling SSL. I am getting "SSL certificate add failed error 1312"

**Reply by nblumhardt** [reply]
Thanks for the follow-up! Unfortunately there's a lot of networking setup that's environment-dependent and outside what we can consider in-scope for Seq.

Regarding SSL setup, that error can indicate one of a few conditions, there are quite a few things to check but thankfully it's almost always one of the possibilities documented in:

https://stackoverflow.com/questions/13076915/ssl-certificate-add-failed-when-binding-to-port

(BTW, if you have a supported Seq license and need some more hands-on help, please drop us a line via our support email address.)

Best regards,
Nick
